extensions.rst

```diff
@@ -0,0 +1,15 @@
+.. _extensions:
+
+Updating standard library extension modules
+===========================================
+
+In this section, we could explain how to write a CPython extension with the C language, but the topic can take a complete book.
+
+For this reason, we prefer to give you some links where you can read a very good documentation.
```

Comment:
Suggested wording/grammar tweak:

> For this reason, we instead prefer to provide some links to existing documentation that we recommend.

(Perhaps it would also be worth mentioning core-mentorship here, and linking to the section on communications?)